assembly/NFT/NFT.ts

```diff
@@ -0,0 +1,232 @@
+///////////////////////////////
+////////////////////////////
+////////////////////////
+// WIP allowance, and transferFrom Missing
+////////////////////////
+////////////////////////////
+///////////////////////////////
+
+import {Address, Storage, Context, generateEvent} from '../std';
+import {ByteArray} from '@massalabs/as/assembly';
+const ownerTokenKey: string = 'ownerOf_';
+const counterKey: string = 'Counter';
+const ownerKey: string = 'Owner';
+const baseURIKey: string = 'baseURI';
+const initCounter: u64 = 0;
+
+/**
+ *The NFT's main characteristics
+ */
+
+const _name: string = 'MASSA_NFT';
+const _symbol: string = 'NFT';
+const _maxSupply: string = '10000';
```

Comment:
why do we hard code the max supply? Does it means that the user will not be able to deploy an NFT with a custom max supply?

Reply:
you just have to change it in the NFT contract

Reply:
let's create an issue about this

Reply:
https://github.com/massalabs/massa-as-sdk/issues/32